app/build.gradle

```diff
@@ -1,10 +1,19 @@
 apply plugin: 'com.android.application'
 
+
+def apikeyPropertiesFile = rootProject.file("apikey.properties")
+def apikeyProperties = new Properties()
+apikeyProperties.load(new FileInputStream(apikeyPropertiesFile))
+
 android {
     compileSdkVersion 30
     buildToolsVersion "30.0.1"
 
     defaultConfig {
+        // should correspond to key/value pairs inside the file
+        buildConfigField("String", "CLIENT_KEY", apikeyProperties['CLIENT_KEY'])
```

Comment:
It might be helpful to have a apikey.properties.example file with blank CLIENT_KEY / APP_ID fields, and have this in the project folder. Then when we add instruction in the readme, we can add to fill that in w/ credentials
We should probably add the apikey.properties file to our .gitignore file as well

Reply:
Ah, yes - this is where my limited view of GitHub comes in - I added it somewhere in a different pull request, eek!
Either way, I added it again to this PR - can you let me know if you see it?

Reply:
Perfect! No, you're totally good. The last thing I would do is just rename it apikey.properties.example. 
If you keep it named apikey.properties, that means that if it gets added to the .gitignore, any changes won't be added to git (like if we were to place our Spoontacular API key in the same file. Also, whenever someone pulls down the PR, they'll have to switch back to another branch to grab their api keys if the file gets overwritten

Reply:
Changed to apikey.properties.example and added apikey.properties to gitignore